src/view/error.twig

```diff
@@ -1,6 +1,6 @@
 {% set pageType = 'error' %}
 {% extends "base-template.twig" %}
-{% block title %}: "Error"{% endblock %}
+{% block title %}{{ "Error"| trans }} - {{ GlobalConfig.serviceName }}{% endblock %}
```

Comment:
There does not seem to be a translation string for 'Error':
* I tried with pages like https://localhost/Skosmos/yso/fi/p%C3%A5ge/p23406
* Then again, `grep -C 1 'Error' skosmos_fi.po` didn't show a msg-id that would be just 'Error'

The 'error' msgid should probably be added to the translation templates.